main.go

```diff
@@ -320,44 +330,36 @@ func generateUUID() string {
 
 func main() {
 	// load config as first thing
-	ConfigLoad()
-
-	// config is used here, call after config load
-	go handleMetrics()
-
-	// Cipher suites as defined in stock Go but without 3DES and RC4
-	// https://golang.org/src/crypto/tls/cipher_suites.go
-	var tlsCipherSuites = []uint16{
```

Comment:
These are removed, as the default suites are sane, and maintained in the stdlib. My understanding is that these were originally chosen as a reduced set of suites. Today, the stdlib suites are even more restricted (and safer).